src/components/AddressBook/AddressBookLocal/AddressBookLocal.tsx

```diff
@@ -19,41 +19,45 @@ export const AddressBookLocal: FunctionComponent<AddressBookLocalProps> = ({
   const { addressBook } = useAddressBook();
 
   return (
-    <table className="table">
-      <thead className="table-thead">
-        <tr>
-          <th>Name</th>
-          <td>Address</td>
-          <td>&nbsp;</td>
-          <td>&nbsp;</td>
-        </tr>
-      </thead>
-      <tbody className="table-tbody">
-        {addressBookLocalStatus === AddressBookState.NONE && (
-          <AddressBookTableRowEmpty message="No address found. Try importing a csv template file?" />
-        )}
-        {addressBookLocalStatus === AddressBookState.EMPTY && (
-          <AddressBookTableRowEmpty message="No address found. Try searching again?" />
-        )}
-        {addressBookLocalStatus === AddressBookState.SUCCESS &&
-          localPageResults.map((key) => {
-            const identifier = addressBook[key];
+    <>
+      <table className="table">
+        <thead className="table-thead">
+          <tr className="hidden md:table-row">
+            <th>Name</th>
+            <th>Address</th>
+            <th>&nbsp;</th>
+            <th>&nbsp;</th>
+          </tr>
+        </thead>
+        <tbody className="table-tbody">
```

Comment:
should we drop `table` layout altogether since now is not as relevant, previously we were making use of `table-responsive` horizontal scroll for mobile, now the proposed design is stacking elements

Reply:
ok dropped tables (table, tbody, th, tr) with divs